### ISSUE TITLE
Можно обходить баны на ксеносов через животинку священника

### ISSUE THREAD

**Issue by cool20141** [TauCetiStation/TauCetiClassic#5629]
#### Подробное описание проблемы
По поводу вот этого бана: https://forum.taucetistation.org/t/apellyacziya-germanrus-nebolshoe-nedorazumenie/19092
Я считаю, что должны быть ограничения для духов, чтоб они не могли войти в куклу, на которой бан (Ну, чтоб бан влиял и на обезьян соответствующих рас). Ну или сделать так, чтоб нельзя было из забаненной (У которой хвост оторвали) макаки через генетику сделать полноценную куклу. Не знаю, пропосал ли это или просто багрепорт, пусть будет второе
#### Что должно было произойти
Нельзя обходить бан таким способом, не дает вселиться в куклу/кукла не становиться высшей своей формой в генетике
#### Что произошло на самом деле
Обход бана на ксенорасу
#### Как повторить
Зайти за животинку, напроситься в генетику

**Comment by KIBORG04** [comment]
А причем тут священник?

**Comment by cool20141** [comment]
Он же создает животинку, в которую можно вселиться

**Comment by Shudyn** [comment]
Так-то можно и просто через генетика мозг пересадить в куклу-ксеноса.

**Comment by cool20141** [comment]
Можно, все можно, только у админов другой взгляд на ситуацию

**Comment by Pivo24** [comment]
Священник спавнит фамильяра в которого можно вселиться, и фамильяр выходит рандомной зверушкой, как я понял из своего опыта игры за котика.

**Comment by AzzyDreemurr13** [comment]
Это все же больше пропосал, поменял тип ишуя
